src/lib/pickles/wrap.ml

```diff
@@ -177,8 +177,23 @@ let deferred_values (type n) ~(sgs : (Backend.Tick.Curve.Affine.t, n) Vector.t)
       ~domain_generator:Backend.Tick.Field.domain_generator
   in
   let tick_env =
+    let module Env_bool = struct
+      type t = bool
+
+      let true_ = true
+
+      let false_ = false
+    end in
```

Comment:
This is like the Hydra: when you read one, 2 others are added in the modules to come 😆

Reply:
Oh, I was thinking the exact same